content/en/examples/access/image-matches-namespace-environment.policy.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "image-matches-namespace-environment.policy.example.com"
```

Comment:
optionally, add a YAML comment or a description annotation to summarize what this policy does.

Reply:
Good point! Added.